pkgs/applications/networking/instant-messengers/slack/default.nix

```diff
@@ -174,7 +182,8 @@ let
     installPhase = ''
       mkdir -p $out/Applications/Slack.app
       cp -R . $out/Applications/Slack.app
-      /usr/bin/defaults write com.tinyspeck.slackmacgap SlackNoAutoUpdates -bool YES
+    '' + lib.optionalString (!stdenv.isAarch64) ''
```

Comment:
Why this change?

Reply:
as I wrote in "things done"

I had to remove the "defaults write com.tinyspeck.slackmacgap" part from aarch64, otherwise 2021-05-31 08:58:32.751 defaults[36345:252092] Could not write domain com.tinyspeck.slackmacgap; exiting

Reply:
Please put the explanation in the comment why this is needed

Reply:
finally, done (or could I make the comment more explanitory?)

Reply:
I think that's OK